10.21.0/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:10.21.0-alpine3.11
```

Comment:
I'd move these files into the root folder. I don't think we gain anything by having a subfolder

Reply:
Moved 👍